packages/arb-token-bridge-ui/tests/e2e/specs/batchDeposit.cy.ts

```diff
@@ -141,11 +141,7 @@ describe('Batch Deposit', () => {
     context('deposit should complete successfully', () => {
       cy.selectTransactionsPanelTab('settled')
 
-      cy.waitUntil(() => cy.findTransactionInTransactionHistory(txData), {
-        errorMsg: 'Could not find settled ERC20 Batch Deposit transaction',
-        timeout: 60_000,
-        interval: 500
-      })
+      cy.findTransactionInTransactionHistory(txData)
```

Comment:
`waitUntil` doesn't work properly here because it needs to return a result. I've moved timeout to the method itself so we always try for `120_000`